Formula/c/chezmoi.rb

```diff
@@ -28,7 +28,7 @@ def install
     ldflags = %W[
       -s -w
       -X main.version=#{version}
-      -X main.commit=#{tap.user}
+      -X main.commit=#{Utils.git_head}
```

Comment:
Maybe this will work?
```suggestion
      -X main.commit=#{File.read("COMMIT")}
```

Reply:
I think this will break when building chezmoi from HEAD, so a bit more logic is needed.

Reply:
HEAD builds are unsupported so someone who runs one can file a pull request for them